website/docs/cluster-management/getting-started.mdx

```diff
@@ -12,7 +12,7 @@ import BrowserOnly from "@docusaurus/BrowserOnly";
 
 ## Creating your first CAPD Cluster
 
-If you've followed the [Upgrade steps](../installation#weave-gitops-enterprise) in the [Installation guide](../installation) you should have:
+If you've followed the [Upgrade steps](installation.mdx#weave-gitops-enterprise) in the [Installation guide](installation.mdx) you should have:
```

Comment:
@ozamosi interesting, in chrome, if I click the upgrade link it navigates to the correct section. If I open in a new tab however, it just goes to the top of the page. This isn't major, but I wondered if theres likely a quick fix ~or whether you think we should force opening in a new tab~? EDIT: related, I wonder if we should force opening in a new tab, but with it screwing up navigation, probably best to avoid unless resolvable